website/content/plugins/drivers/community/pledge.mdx

```diff
@@ -0,0 +1,333 @@
+---
+layout: docs
+page_title: 'Drivers: Pledge'
+description: The Pledge task driver is capable of blocking unwanted syscall and filesystem access.
+---
+
+# Pledge Driver
+
+Name: `pledge`
+
+The `pledge` driver executes programs similar to `exec` or `raw_exec`, but
+provides sandboxing by restricting syscalls the process is allowed to make.
+The `pledge` driver uses the [Landlock LSM][landlock] to restrict filesystem
+access, which can be significantly more efficient than creating a `chroot`.
+Isolation of CPU and memory resources is provided by leveraging features of
+cgroups v2. Network, PID, and IPC isolation is powered by Linux namespaces.
+
+Source on [GitHub][repository]
+
+Downloads on GitHub [releases][releases]
+
+#### Use cases
+
+The `pledge` driver is intended as a replacement for `raw_exec`. Sometimes
+there are those management tasks that just need to be run as `root` and directly
+access the filesystem or perform privileged operations. While `raw_exec`
+provides no isolation, the `pledge` driver uses Landlock to restrict the files
+or directories the task is allowed to access and modify. Specific groups of
+system calls are allow-listed in the task configuration, greatly reducing the
+attack surface of a mis-configured or compromised task.
+
+#### About
+
+The `pledge` driver is fundamentally powered by the [pledge utility for Linux]
+[pledge] by Justine Tunney. The driver invokes this `pledge.com` CLI tool along
+with `nsenter` and `unshare` to create an attuned sandbox in which to execute a
```

Comment:
Is this supposed to be "attenuated"?

Reply:
Heh no, I was using attuned like "define the sandbox in harmony with the command being run". But that's obviously not that clear, so I'll change the wording.